### PR TITLE
Add SuSAv2 EM QE hadron tensors separated by hit nucleon isospin

### PR DESCRIPTION
**Pull request by LiangLiu212**
After last genie Core developer meeting, I made some changes according to the suggestions. 

The existing implementation of the SuSAv2 EM QE model doesn't distinguish between struck protons and neutrons, leading to an incorrect equal cross section for p and n knockout. Liang Liu (@LiangLiu212) has added tensors from Guillermo Megias separated by isospin, together with code to handle them correctly for cross section calculations and event generation.

### CODE REVIEW

File: do_configure.sh
Comment (mroda88) [inline do_configure.sh:1]: This file to be removed from git

File: config/SuSAv2QELPXSec.xml
Comment (mroda88) [inline config/SuSAv2QELPXSec.xml:1]: Can you restore the indentation to improve readability?

**Review by mroda88 — CHANGES_REQUESTED** [review]
The tune should disappear, we should use the standard SuSAv2 CMC that are the G21_11*

File: do_configure.sh
Comment (LiangLiu212) [reply do_configure.sh:1]: This file has been removed.

File: config/SuSAv2QELPXSec.xml
Comment (LiangLiu212) [reply config/SuSAv2QELPXSec.xml:1]: This configuration has been restored from master branch.

**Pull request closed without merging**